### PR TITLE
Color binding changes

### PR DESCRIPTION
**Pull request by atsushieno**
let's see how generator is messing around by simply enabling color metadata fixup. Things are not as simple and stupid as you dream without working on it...

### CODE REVIEW

**Comment by atsushieno** [comment]
For internal reference: original Color design was added back in 2012 at monodroid/e47da17f

**Comment by atsushieno** [comment]
regarding https://github.com/xamarin/xamarin-android/issues/1201 here is the results from `mono --debug /sources/xamarin-android/bin/BuildDebug/mono-api-html.exe reference/Mono.Android.xml temp/Mono.Android.xml --ignore-changes-parameter-names ` at `external/xamarin-android-api-compatibility`:

 <div>
<h3>Type Changed: Android.App.WallpaperManager</h3>
<div>
<p>Added methods:</p>
<pre>
	<span class='added added-method ' data-is-non-breaking>public virtual void AddOnColorsChangedListener (WallpaperManager.IOnColorsChangedListener, Android.OS.Handler);</span>
	<span class='added added-method ' data-is-non-breaking>public virtual WallpaperColors GetWallpaperColors (int);</span>
	<span class='added added-method ' data-is-non-breaking>public virtual void RemoveOnColorsChangedListener (WallpaperManager.IOnColorsChangedListener);</span>
</pre>
</div>

File: src/Mono.Android/metadata [hunk 1]
```diff
@@ -1390,16 +1387,10 @@
 
   We will have to enable this line and bring back some sanity.
   -->
-  <!-- attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">AndroidColor</attr -->
+  <attr path="/api/package[@name='android.graphics']/class[@name='Color']" name="managedName">ColorObject</attr>
```
Comment (jonpryor) [inline src/Mono.Android/metadata]: Why `ColorObject` and not `AndroidColor`?

Comment (atsushieno) [reply src/Mono.Android/metadata]: I am 100% sure that people try to look up "Color" something. AndroidColor will never give that impression to developers.

**Pull request merged**